cmd/registry-graphql/graphql/specs.go

```diff
@@ -63,14 +62,11 @@ var specType = graphql.NewObject(
 			"updated": &graphql.Field{
 				Type: timestampType,
 			},
-			"contents": &graphql.Field{
```

Comment:
This removes the "contents" field from responses in the GraphQL demo. This is the only reduction in functionality in this PR.